osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -269,9 +269,6 @@ public override void OnEntering(ScreenTransitionEvent e)
                 dialogOverlay?.Push(new StorageErrorDialog(osuStorage, osuStorage.Error));
         }
 
-        [CanBeNull]
```

Comment:
These changes can be dropped if preferred, it just weirded me out that we're storing the local for no good reason.